payloads/src/external/simple/testtesttest.S

```diff
@@ -36,8 +36,7 @@ _start:
 
 	call check_full
 	li t1, '0'
-	csrr t2, mhartid
-	add t1, t1, t2
+	add t1, t1, a0
```

Comment:
I believe in coreboot the decision was made not to trust a0. Sure you want to do that? Fine with me if so.

Reply:
This is a test payload. The problem is when the payload is run in S-mode, it cannot access mhartid CSR.